main.js

```diff
@@ -0,0 +1,54 @@
+// DOM utility functions
+const addClass = (el, newClass) => {
+  el.className += ` ${newClass}`;
+};
+
+const removeClass = (el, oldClass) => {
```

Comment:
Question - why `const` here instead of `let`? Not that I'm doubting the REASON for `const`, I actually just don't know the logic and would love to learn :grin: